frameworks/hdfs/src/test/java/com/mesosphere/sdk/hdfs/scheduler/HDFSUserAuthMapperBuilderTest.java

```diff
@@ -60,6 +60,17 @@ public void testAddDefaultUserAuthMapping() {
         Assert.assertEquals(expected, authMappings);
     }
 
+    @Test
+    public void testEnvAuthToLocalIsNotOverwritten() {
+        String expected = "authToLocal";
+        Map<String, String> env = getEnvWithEncodedAuthToLocal(expected);
+        String authMappings = new HDFSUserAuthMapperBuilder(env, frameworkHost)
+                .addUserAuthMappingFromEnv()
+                .addDefaultUserAuthMapping("data", "node", 1)
+                .build();
+        Assert.assertEquals(expected, authMappings.split("\n")[0]);
```

Comment:
Minor point: This implies a particular ordering. We could do a search in the lines instead.